### ISSUE TITLE
Config file

### ISSUE THREAD

**Issue by nicovs** [ltanyc/coinStats#4]
hey @ltanyc, 

can you add a config file in here? (no JS guru here :D)

makes it easier to declare the Explorer API or InfluxDB host and DB name

**Comment by ltanyc** [comment]
yeah, a JSON config file similar to the following format would be nice to add:

{
 influxdbHost: 'localhost',
 influxdbName: 'coinStats',
 coins: [
  { coinName: 'coin1', explorerApi: 'url1', cmcApi: 'cmc_url', ...},
  { coinName: 'coin2', explorerApi: 'url2', poolApi: 'pool_url', ...},
  ...
 ]
}

Not sure when I can get to this if someone wants to try it before I can do it lmk and I can help...

**Comment by Bhanderi-Aksahy** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @zzir, @adityasakhuja, @yolio2003, @Akshit-Soneji, @gvlax, @Matt-Jensen, @unitycoder, @aistrych, @Balou9, @ShowBaba, @haai, @Ir1d, @Phonbopit, @ahmgeek, @floramica, @darky, @RaviGuptaa, @sirakoff, @sorin89, @mfer725
